assets/docs/gatewayparameters.md

```diff
@@ -1,9 +1,9 @@
-GatewayParameters is a kgateway custom resource that specifies the configuration for the gateway proxies in your cluster. 
+{{< reuse "docs/snippets/gatewayparameters.md" >}} is a custom resource that specifies the configuration for the gateway proxies in your cluster. 
 
-To spin up new gateway proxies, the kgateway controller uses a [gateway proxy template](#gatewayproxytemplate) and the configuration in the default GatewayParameters resource. You can change the default configuration for your gateway proxies by updating the respective values in the kgateway Helm chart. You can also create additional GatewayParameters resources to spin up different types of gateway proxies.
+To spin up new gateway proxies, the {{< reuse "docs/snippets/kgateway.md" >}} controller uses a [gateway proxy template](#gateway-proxy-template) and the configuration in the default {{< reuse "docs/snippets/gatewayparameters.md" >}} resource. You can change the default configuration for your gateway proxies by updating the respective values in the {{< reuse "docs/snippets/kgateway.md" >}} Helm chart. You can also create additional {{< reuse "docs/snippets/gatewayparameters.md" >}} resources to spin up different types of gateway proxies.
 
 {{< callout type="info" >}}
-Do not edit or change the default GatewayParameters resource directly. Always update the values in the kgateway Helm chart so that they persist between upgrades, or create your own GatewayParameters resource that holds your custom values. Note that you are responsible to maintain any GatewayParameters resources that you manually created.  
+Do not edit or change the default {{< reuse "docs/snippets/gatewayparameters.md" >}} resource directly. Always update the values in the {{< reuse "docs/snippets/kgateway.md" >}} Helm chart so that they persist between upgrades, or create your own {{< reuse "docs/snippets/gatewayparameters.md" >}} resource that holds your custom values. Note that you are responsible to maintain any {{< reuse "docs/snippets/gatewayparameters.md" >}} resources that you manually created.  
```

Comment:
not in this PR, but we might need to figure out what is going on with GatewayParameters in GGv2 to decide how to reuse this content. I don't see a default one when I run `kubectl get gatewayparameters -A` or `kubectl get gloogatewayparameters -A`